pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -3750,13 +3751,14 @@ jobs:
     - get: ops-ui-master
       trigger: true
       passed: ["WL Terraform"]
-    - task: "WL Helm"
-      file: census-rm-deploy/tasks/helm.yml
+    - task: "WL Rabbitmq"
+      file: ((rabbitmq-file))
       params:
         ADMIN_SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
         ENV: whitelodge
         KUBERNETES_CLUSTER: rm-k8s-cluster
         RABBITMQ_CONFIG_VALUES_FILE: rabbitmq/values.yml
```

Comment:
`RABBITMQ_CONFIG_VALUES_FILE` is being passed in but rabbitmq-provision.yml isn't using it.